src/send/SendConfirmation.tsx

```diff
@@ -260,10 +260,7 @@ function SendConfirmation(props: Props) {
   const allowComment = !isInvite && isStablecoin(tokenInfo)
 
   return (
-    <SafeAreaView
-      style={styles.container}
-      edges={props.route.name === Screens.SendConfirmationModal ? ['bottom'] : undefined}
```

Comment:
why remove this line?

Reply:
This was causing the issue with Send confirmation Screen seen in @jeanregisser screenshot, as it was only happening on the `SendConfirmation.tsx` screen and `SendConfrimationLegacy.tsx`. Since it's working with the defaults I opted to use those instead of passing undefined.

Reply:
Fixed with downgrade in 5d947ba8bb9c3a3f0740a60a59c04fa33e52a533.